.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -26,6 +31,13 @@ assignees: ''
 <!-- for the talk i.e. software, tools set up on machine etc, please -->
 <!-- include it here -->
 
+#### Speaker bio
```

Comment:
How about we only ask for Speaker bio here instead of social media links? Is it possible to do all the communication on proposal issue discussion itself?

Reply:
No it's not. Again from past experience, 
1. not everyone is active on GitHub all the time. We have had instances where we had to find the persons contact details with help of common contacts, their colleagues.
2. All the communication will happen on GitHub issue itself as it happens right now, the details are for situations like above and to share the private Jitsi Meet link and the password (not a good idea to share that publicly).
3. It's good to have LinkedIn or Twitter handles so that other attendees can contact the speakers in case they have some doubts etc. 

As written there it's not compulsory to share the links, they can contact via mail as well, that way their details will remain private.

Reply:
And we are asking for bio here along with links, let me make that explicit by mentioning it.